morphuntion/src/icu4cxx/BreakIterator.hpp

```diff
@@ -0,0 +1,126 @@
+/*
+ * Copyright 2016-2024 Apple Inc. All rights reserved.
+ */
+// This is a generated file. Do not edit this file.
```

Comment:
Do we care how it's generated? If yes, some epointers to the generator would be good.

Reply:
This is generated with the generateICUWrapper tool.  It takes an existing ICU installation, and generates C++ wrappers around the C API.  This provides a natural adaptor for C++ API, but with the stability of the C ABI.